eks/terraform.tfvars

```diff
@@ -2,8 +2,53 @@
 # SPDX-License-Identifier: Apache-2.0
 
 # Sample tfvars file. Uncomment out values to use
+# Do not commit this file to Git with sensitive values
+
 # cluster_name = "holoscan-cluster"
 # region  = "us-west-2"
 
 # Optional: If deploying into an existing VPC, use the following variable
-# existing_vpc_details = {vpc_id = "", subnet_ids = ["", ""]}
+# additional_node_security_groups_rules = {}
+# additional_security_group_ids         = []
+# additional_user_data                  = ""
+# aws_profile                           = "development"
+# cidr_block                            = "10.0.0.0/16"
+# cluster_name                          = ""
```

Comment:
Already have cluster_name above

Reply:
Fixed in !11

Reply:
Caught this after the merge. Fixed in #11